data/net/minecraft/client/gui/Font.mapping

```diff
@@ -21,108 +30,144 @@ CLASS net/minecraft/client/gui/Font
 		ARG 2 x
 		ARG 3 y
 		ARG 4 color
-		ARG 5 dropShadow
+		ARG 5 drawShadow
 		ARG 6 matrix
 		ARG 7 buffer
-		ARG 8 transparent
+		ARG 8 seeThrough
```

Comment:
I did it because eventually it leads down to the constructor of StringRenderOutput where it decides if the display mode should be SEE_THROUGH or NORMAL. I can of course change it back though!

Reply:
In which case, matching mojang's name would be better here imo